types/titanium/.eslintrc.json

```diff
@@ -0,0 +1,7 @@
+{
+    "rules": {
+        "no-any-union": "off",
+        "no-unnecessary-generics": "off",
+        "prefer-declare-function": "off"
```

Comment:
Can you provide some examples/rationale as to why these need to be disabled? Each of these should be flagging real issues.

Reply:
the test showed an error for every file:
```
/DefinitelyTyped/types/titanium/base.d.ts
  1:1  error  Definition for rule 'no-any-union' was not found             no-any-union
  1:1  error  Definition for rule 'no-unnecessary-generics' was not found  no-unnecessary-generics
  1:1  error  Definition for rule 'prefer-declare-function' was not found  prefer-declare-function
```
so I've created that file

Reply:
In CI? Or locally? It seems to me like you just needed to update main and update your copy of dtslint.

Reply:
locally, when I run the last step in the todo list.
But you are right! Updating master and running the test again without the file worked fine too :+1:  Thanks for the hint